bluesky/callbacks/core.py

```diff
@@ -534,13 +534,17 @@ def start(self, doc):
         self.im = im
         self.ax.set_title('scan {uid} [{sid}]'.format(sid=doc['scan_id'],
                                                       uid=doc['uid'][:6]))
+        self.snaking = doc.get('snaking', (False, False))
```

Comment:
:100: 

When I write code that I can't even recall, I know I'm on my way to @tacaswell levels of glory. (Thanks for obliging me by digging this up.)